Foundation/NSCharacterSet.swift

```diff
@@ -168,6 +168,10 @@ public class NSCharacterSet : NSObject, NSCopying, NSMutableCopying, NSCoding {
         return CFCharacterSetCreateCopy(kCFAllocatorSystemDefault, self._cfObject)
     }
     
+    public override func mutableCopy() -> AnyObject {
```

Comment:
I've got a problem with `as?` statements on linux. I already noticed it twice: in `_NSObjectRepresentableBridge` function and now in `NSObject`'s `mutableCopy`:

```
        if let copyable = self as? NSMutableCopying {
```

this condition passes on OS X and fails on linux.

This override is a workaround for this problem.